invenio_rdm_pure/source/utils.py

```diff
@@ -24,6 +24,15 @@
 )
 
 
+def load_file_as_string(path):
```

Comment:
could you please add some error handling in the case the file at the path does not exists

Reply:
Error handling added.